currencies/rates_test.go

```diff
@@ -163,6 +163,39 @@ func TestGetRate(t *testing.T) {
 	}
 }
 
+func TestGetRate_ReverseConversion(t *testing.T) {
+
+	// Setup:
+	rates := currencies.NewRates(time.Now(), map[string]map[string]float64{
+		"USD": {
+			"GBP": 0.77208,
+		},
+		"EUR": {
+			"USD": 0.88723,
+		},
+	})
+
+	testCases := []struct {
+		from         string
+		to           string
+		expectedRate float64
+	}{
+		{from: "USD", to: "GBP", expectedRate: 0.77208},
```

Comment:
Makes sense to me and looks great. Could you please add a description to the test and pass it to the asserts so if a test case fails, we can know which one specifically and to explicitly call out which conversions are direct vs "reverse engineered".

Reply:
Hey ! Yes sure, just added, thanks :)